types/axios-curlirize/index.d.ts

```diff
@@ -5,12 +5,16 @@
 
 export = AxiosCurlirize;
```

Comment:
Normally, this would have been fast-tracked, since you're the owner.  However, it looks like the header template wasn't updated, so the bot couldn't identify you.  Was that supposed to be https://github.com/stevenhankin?

Reply:
Hello @amcasey !  Yes, I'm the owner of the template (but not of the referenced axios-curlirize module)

Reply:
Sorry, I think my question was unclear.  At the top of this file, it says 
```ts
// Definitions by: Steven Hankin <https://github.com/me>
```
Should it instead say
```ts
// Definitions by: Steven Hankin <https://github.com/stevenhankin>
```

Reply:
Lol...sorry, I'm new to this stuff.  Yes it should; I'll update the PR in a few mins...

Reply:
No worries.  Once you correct it, future changes you make to this type definition should be fast-tracked by the bot.